sway/container.c

```diff
@@ -19,6 +19,47 @@
 	sway_assert (PTR, #PTR "must be non-null")
 
 
+struct containers_by_handle_element {
+	wlc_handle handle;
+	swayc_t *cont;
+};
+
+static int compare_wlc_handle_qsort(const void *handle1, const void *handle2) {
+	return (*(const struct containers_by_handle_element**)handle1)->handle - (*(const struct containers_by_handle_element**)handle2)->handle;
+}
+
+static int compare_wlc_handle_bsearch(const void *handle1, const void *handle2) {
+    return ((struct containers_by_handle_element*)handle1)->handle - (*(const struct containers_by_handle_element**)handle2)->handle;
+}
```

Comment:
Can you add more newlines in these functions? Try to limit your code to 80 characters wide.